granulate_utils/containers/client.py

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+import logging
+from typing import List
+
+from granulate_utils.containers.container import Container
+from granulate_utils.containers.cri.client import CRIClient
+from granulate_utils.containers.docker import DockerClient
+
+
+class ContainersClient:
+    """
+    Wraps DockerClient and CRIClient to provide a unified view of all containers
+    running on a system.
+    """
+
+    def __init__(self):
+        try:
+            self._docker_client = DockerClient()
+        except Exception:
+            logging.warning("Failed to initialize DockerClient, skipping...", exc_info=True)
```

Comment:
We don't use logging yet here. Decide how it should cooperate with gprofiler's logging